e2e-polybft/e2e/broadcast_test.go

```diff
@@ -0,0 +1,91 @@
+package e2e
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/wallet"
+
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+// TestE2E_Broadcast sends several transactions (legacy and dynamic fees) to the cluster with the 1 amount of eth
+// and checks that all cluster nodes have the recipient balance updated.
+func TestE2E_Broadcast(t *testing.T) {
+	t.Parallel()
+
+	const (
+		sendAmount = int64(1)
```

Comment:
WDYT? Since this value is not expressed in tokens, but instead in WEIs. So in order for the test to be in accordance with test description, suggesting this change.
```suggestion
		sendAmount = ethgo.Ether(1)
```

Reply:
Done